pyproject.toml

```diff
@@ -159,7 +157,6 @@ show_missing = true
 
 [tool.codespell]
 skip = '*.pyc,*.txt,*.gif,*.png,*.jpg,*.js,*.html,*.doctree,*.ttf,*.woff,*.woff2,*.eot,*.mp4,*.inv,*.pickle,*.ipynb,flycheck*,./.git/*,./.hypothesis/*,*.yml,./doc/build/*,./doc/images/*,./dist/*,*~,.hypothesis*,./doc/source/examples/*,*cover,*.dat,*.mac,*.cdb,*.CDB,build,./docker/mapdl/v*,./factory/*,./ansys/mapdl/core/mapdl_functions.py,PKG-INFO,*.mypy_cache/*,./docker/mapdl/*,./_unused/*'
-ignore-words = "doc/styles/config/vocabularies/ANSYS/accept.txt"
```

Comment:
Why removing this line?

Reply:
Some time ago, I faced a situation in which codespell, when called via `pre-commit`, did not respect the configuration declared in `pyproject.toml`. Even after installing the `tomli` deps.

Ideally, all the configuration should live inside the `pyproject.toml`, which is the official Python file. pre-commit should just execute the program, which must read then the `pyproject.toml` file.

Reply:
ok thanks @jorgepiloto , noted and reverted the changes.